src/c++11/tracer.h

```diff
@@ -47,6 +43,24 @@ class Tracer {
   // Get the implementation object.
   ImplPtr impl() const { return impl_; }
 
+  // Inject() takes the `sc` SpanContext instance and injects it for
+  // propagation within `carrier`. The actual type of `carrier` depends on
+  // the value of `format`.
+  //
+  // OpenTracing defines a common set of `format` values (see BuiltinFormat),
+  // and each has an expected carrier type.
+  //
+  // TODO throws?
```

Comment:
C++ exceptions make me nervous... return a bool and take an optional output param for the error??